config.go

```diff
@@ -194,4 +194,30 @@ func continueTask(s string, def string) (cont bool) {
 	}
 
 	return true
-}
\ No newline at end of file
+}
+
+// PassToPacman outsorces execution to pacman binary without modifications.
+func passToPacman(op string, pkgs []string, flags []string) error {
```

Comment:
Looks like you merged this incorrectly. This function was moved but your branch has both. Have you actually checked your branch compiles? It does not compile for me at least.

Reply:
Sorry let me check it again